### DISCUSSION TITLE
Custom path for index download in installation sample code

### DISCUSSION THREAD

**Discussion by sdan2** [castorini/pyserini#1839] [Q&A] [answered]
Hi, when I run 
`python -m pyserini.search.lucene \
    --topics msmarco-passage-dev-subset \
    --index msmarco-v1-passage \
    --output run.msmarco-passage.txt \
    --output-format msmarco \
    --bm25`
it downloads the 2 GB indices to my home directory.
How do I set a custom download location for the large index files?

**Comment by lintool** [comment, marked as answer]
Use the `PYSERINI_CACHE` env variable, e.g.,

```
export PYSERINI_CACHE=/my/path/here
```
